docs/metrics/rouge-n.md

```diff
@@ -10,7 +10,7 @@ metric collection, is a vital metric in the field of natural language processing
 quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts. ROUGE-N
 provides insights into the ability of a system to capture essential content and linguistic nuances, making it an
 important and versatile tool used in many NLP workflows. As the name implies, it is a recall-based metric — a complement
-to the precision-based BLEU score.
+to the precision-based BLEU score. ROUGE-N ranges between 0 and 1, with 0 indicating no overlap and 1 indicating a perfect overlap.
```

Comment:
I personally think it fits better after "It assesses the
quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts." this line. What do you think?

Reply:
Yup, that does flow better than what I had previously. I've rearranged the first paragraph to include your comment.